client/jetpack-connect/test/__snapshots__/signup.js.snap

```diff
@@ -500,8 +499,7 @@ exports[`JetpackSignup should render with locale suggestions 1`] = `
                 .
               </p>
               <button
-                class="button signup-form__submit form-button is-primary"
-                locale="en"
```

Comment:
If this test is meant to test that `JetpackSignup should render with locale suggestions` and we've removed the `locale`, shouldn't we also delete the test?

(I'm not sure this was working correctly to start with as the test sets `locale="es"` and this snapshot had `locale="en"`)

Reply:
Good question. I believe the locale attribute on the button isn't actually used or related to the test. I ran a diff on the snapshot output and the `locale="es"` in the test props results in a different signup URL:

`href="/log-in/jetpack/es?site=http%3A%2F%2Fan.example.site&redirect_to=https%3A%2F%2Fexample.com%2F&email_address=email%40an.example.site&from=banner-44-slide-1-dashboard"`

vs

`href="/log-in/jetpack?site=http%3A%2F%2Fan.example.site&redirect_to=https%3A%2F%2Fexample.com%2F&email_address=email%40an.example.site&from=banner-44-slide-1-dashboard"`

So I think it's still valid.

Reply:
Oh that makes sense!